internal/httputil/certlog.go

```diff
@@ -0,0 +1,157 @@
+package httputil
```

Comment:
it kind if weird that the certlog is in httputil pkg. Should we keep outside of httputil? Same comment for internal/httputil/certutil.go. 

It is nit pick and not blocker for this PR.